LEAF_Request_Portal/api/controllers/SignaturesController.php

```diff
@@ -20,12 +23,22 @@ public function __construct($db, $login)
 
     public function get($action)
     {
+        $signature = $this->signature;
+
         $this->index['GET'] = new ControllerMap();
 
         $this->index['GET']->register('signature/version', function () {
             return $this->API_VERSION;
         });
 
+        $this->index['GET']->register('signature/[digit]', function ($args) use ($signature) {
+            return $signature->getSignature((int)$args[0]);
+        });
+
+        $this->index['GET']->register('signature/[digit]/history', function ($args) use ($signature) {
+            return $signature->getSignatureHistory((int)$args[0]);
+        });
```

Comment:
The controller endpoints `signature/[digit]` and `signature/[digit]/history` should be moved to `FormController` to reflect the changes made in the `Signature` class methods, since these really operate on a record instead of a signature. However the methods below should stay in `Signatures.php`.